### PR TITLE
Update terms_and_abbreviations.tex

### PR DESCRIPTION
**Pull request by ardell96**

### CODE REVIEW

File: docs/design/terms_and_abbreviations.tex [hunk 1]
```diff
@@ -2,4 +2,50 @@
 
 \section*{Terms and Abbreviations}
 
+\begin{tabular}[h]{rp{0.75\linewidth}}
+    \hline
+    \textbf{Term} & \textbf{Definition}\\
+    \hline
+
+    ANN & Artificial Neural Network, or simply ``Neural Network'', is a data processing model modeled after neuron interactions. The process consists of forward propagation through the use of several matrix multiplications.\cite{ann}\\
```
Comment (peterdeutsch) [inline docs/design/terms_and_abbreviations.tex]: Remove

File: docs/design/terms_and_abbreviations.tex [hunk 2]
```diff
@@ -2,4 +2,50 @@
...
+    FPS & Frames Per Second.\\
+    GPU & Graphics Processing Unit, a discrete piece of hardware designed to accelerate graphics-intensive or other parallel computing tasks.\\
+    HPS & Hard Processor System. \\
+    IC & Integrated Circuit. \\
+	ICICS & Institute for Computing, Information, and Cognitive Systems at the University of British Columbia.\\
+    IEEE & Institute of Electrical and Electronics Engineers. \\
+    I/O & Input / Output. \\
+    LC & Logic Cell. \\
+    LOS & Line-of-sight.\\
+    Marlann & Multiply-Accumulate and Rectified-Linear Accelerator for Neural Networks. \\
```
Comment (peterdeutsch) [inline docs/design/terms_and_abbreviations.tex]:
```suggestion
    MARLANN & Multiply-Accumulate and Rectified-Linear Accelerator for Neural Networks, an open-source IP block. \\
```

File: docs/design/terms_and_abbreviations.tex [hunk 3]
```diff
@@ -2,4 +2,50 @@
...
+    CMOS & Complimentary Metal Oxide Semiconductor: a type of digital logic. \\
+    CNN & Convolutional Neural Networks are neural networks which are especially useful for image classification.\cite{cnn} \\
+    CPU & Central Processing Unit. \\
+    DNN & Deep Neural Network. \\
+    ECE & Department of Electrical and Computer Engineering at the University of British Columbia.\\
+    FPGA & Field-Programmable-Gate-Array, a ``programmable'' hardware unit that allows for ASIC-like performance with software-like turn-around time and flexibility.\\
+    FPS & Frames Per Second.\\
+    GPU & Graphics Processing Unit, a discrete piece of hardware designed to accelerate graphics-intensive or other parallel computing tasks.\\
+    HPS & Hard Processor System. \\
+    IC & Integrated Circuit. \\
```
Comment (peterdeutsch) [inline docs/design/terms_and_abbreviations.tex]: Add IP (Intellectual Property)

File: docs/design/terms_and_abbreviations.tex (hunk 1 above)
Comment (ardell96) [reply docs/design/terms_and_abbreviations.tex]: why do you want to remove ANN?

Comment (peterdeutsch) [reply docs/design/terms_and_abbreviations.tex]: I don't think it's used in the document, is it?

Comment (ardell96) [reply docs/design/terms_and_abbreviations.tex]: I didn't think that that mattered, but I'll remove it.

**Pull request merged**